### ISSUE TITLE
Determine Data Provenance 

### ISSUE THREAD

**Issue by ekivemark** [HHSIDEAlab/pjson#1]
This JSON format has great potential. I feel that it needs two additional elements:
1. Date of last update.
2. Author

These two data elements in conjunction with other data about the source/location of the record can help third party applications to assess the provenance of the data. 

The date of last update will allow record comparisons to determine the newest published record.

The Author data when coupled with the source that the record was downloaded from will help third party applications to assess the integrity of the record. For example. A record authored by a provider and downloaded from the providers web site ought to be considered to have greater authenticity than an association record downloaded from a third-party site. 

Author information would also help to trace back to the originator of the information. 

I am not advocating a complex authentication protocol be established in order to attribute author information. I am simply advocating author information be included that can be cross-checked with, for example, the URL source of the data to determine authenticity. This goes towards enabling the Association record format to be de-centralized and then found and compiled by search engines etc. 

The combination of author and source will still allow directory services to be created by third party organizations. Consumers of Association data can develop white/black lists of sources/URLs from which this data can be compiled or excluded.

**Comment by aviars** [comment]
Hi Mark:

Thats for being the first person to comment on this via this issue tracker.  I hope your actions inspire others. 

Let me address your items.

1. Last update date - this is in there but was not shown for simplicity in this document.  its in the ... and is called "last_updated_epoch" .

2. This is currently missing from the specification and I would agree with your critique that it should be added.  I will close this issue  when I add this.  Should it just be a text field?

**Comment by ekivemark** [comment]
Yes I think author field should at this stage just be text - unless you apply same format controls used for the end points for the associations themselves. Eg, direct address, URL etc.

Mark Scrimshire
Health and Cloud Technologist
http://ekivemark.com
I/S/T: @ekivemark
C:+1.703.623.2789
